### DISCUSSION TITLE
Capes

### DISCUSSION THREAD

**Discussion by johron** [BleachDrinker420/BleachHack#342] [Ideas]
# Add a cape module
- Everyone with bleachhack can se it
- change the cape file in the /bleach/ folder
- make it cool

**Comment by nhtoTheMonke** [comment]
fortnite

**Comment by BleachDev** [comment]
can't wait for someone to upload a 5gb cape

**Comment by ChiquitaV2** [comment]
You could have it work with the online system, so that everyone running bleach hack would get a cape.

**Reply by GDBrallon** [reply]
why tf do people want capes so badly, such a useless thing to add

**Reply by nhtoTheMonke** [reply]
cause capes cool

**Comment by RedCarlos26** [comment]
That would be very cool fr
